docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +106,255 @@ To manage file storages:
- - You can copy and move files and directories within the virtual file system with the `copyTo()` and `moveTo()` methods.If a file or directory of the same name already exists in the target location, the overwrite input parameter of the method defines whether the existing file is overwritten.
-	> **Note**
-	> The file or directory to be copied or moved must be located under the current directory.
+- You can read and write to a file by first using the `openFile()` method to open the file. You can specify the file mode.The `openFile()` method returns a `FileHandle` object (in [mobile](../../api/latest/device_api/mobile/tizen/filesystem.html#FileHandle), [wearable](../../api/latest/device_api/wearable/tizen/filesystem.html#FileHandle), and [TV](../../api/latest/device_api/tv/tizen/filesystem.html#FileHandle) applications), which is a handle to the opened file. All actual operations, such as read, write, or close, on the file are performed through the `FileHandle` object based on a curent position, which can be obtained or modified through calling `seek()` operation on `FileHandle` object.
+
+To write to files:
 
-To read and write to files, and move and copy files and directories:
+1. To open or create a file, use the `openFile()` method of the `FileSystemManager` interface (in [mobile](../../api/latest/device_api/mobile/tizen/filesystem.html#FileSystemManager), [wearable](../../api/latest/device_api/wearable/tizen/filesystem.html#FileSystemManager), and [TV](../../api/latest/device_api/tv/tizen/filesystem.html#FileSystemManager) applications).
 
-1. To open a file, use the `openStream()` method of the `File` interface (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/filesystem.html#File), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/filesystem.html#File), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/filesystem.html#File) applications).
+   The method returns a `FileHandle` object, which is a handle to the opened or newly created file.
 
-   The method returns a `FileStream` object, which is a handle to the opened file.
+   > **Note**
+   > openFile() method used with mode 'w' creates new file or opens existing file and erases it's contents. Any data being previously in a file opened in that mode will be lost.
```

Comment:
openFile() method used with mode 'w' creates new file or opens existing file and erases it's contents. Any data contained in a file opened in that mode will be lost.